docs/ops/cli.md

```diff
@@ -138,17 +138,30 @@ This allows the job to finish processing all inflight data.
 
 [Savepoints]({{site.baseurl}}/ops/state/savepoints.html) are controlled via the command line client:
 
-#### Trigger a savepoint
+#### Trigger a Savepoint
 
 {% highlight bash %}
-./bin/flink savepoint <jobID> [savepointDirectory]
+./bin/flink savepoint <jobId> [savepointDirectory]
 {% endhighlight %}
 
-Returns the path of the created savepoint. You need this path to restore and dispose savepoints.
+This will trigger a savepoint for the job with ID `jobId`, and returns the path of the created savepoint. You need this path to restore and dispose savepoints.
 
-You can optionally specify a `savepointDirectory` when triggering the savepoint. If you don't specify one here, you need to configure a default savepoint directory for the Flink installation (see [Savepoints]({{site.baseurl}}/ops/state/savepoints.html#configuration)).
 
-##### Cancel with a savepoint
+Furthermore, you can optionally specify a target file system directory to store the savepoint in. The directory needs to be accessible by the JobManager.
```

Comment:
what do you by 'specifying the target path'? AFAIU, it has to be a directory, because the checkpoint dir name is a random hash.

The wordage of this part is borrowed from [the original doc](https://ci.apache.org/projects/flink/flink-docs-release-1.3/setup/cli.html#trigger-a-savepoint)